### ISSUE TITLE
I would like a way to see all recent notifications for me in a channel

### ISSUE THREAD

**Issue by setharnold** [matterhorn-chat/matterhorn#786]
Hello, I recently returned from a two week vacation. (Strongly recommended.)

There's four notifications for me somewhere in a channel with thousands of messages. Normally I try to skim all the scrollback in my 'important' channels each morning but this is daunting after a long vacation. (It'll even take a long time to request all the messages via the "load older messages" tool.)

In irssi, I can use the `/lastlog -hilight` command to get a very quick display of all the messages that triggered any of my highlights. (irssi's `/lastlog` command can also search for any substrings in messages and show only those (similar to `grep` or vi `:g`).)

What I would love is an implementation of something like irssi's `/lastlog`. What would help me immensely right now would be something like irssi's `/lastlog -hilight`.

Thanks

**Comment by 3v1n0** [comment]
It's not too different from https://github.com/matterhorn-chat/matterhorn/issues/620 I think.

**Comment by setharnold** [comment]
@3v1n0 yeah, I thought about that, but my conversation about irssi's `/lastlog` on #620 may have derailed that issue too far :( -- so I thought I'd put my specific desire in a new issue, so #620 can remain focused on feature parity with the web application. (Obviously, if the matterhorn team feels it's similar enough to just close this, that's fine by me, too!)

Thanks
